### ISSUE TITLE
University of New Mexico Earth Sciences

### ISSUE THREAD

**Issue by ccicero** [ArctosDB/new-collections#13]
[Univ_NM_Earth_Sciences_20170623.pdf](https://github.com/ArctosDB/new-collections/files/1099957/Univ_NM_Earth_Sciences_20170623.pdf)

**Comment by campmlc** [comment]
Waiting on GRBIO registration.

**Comment by campmlc** [comment]
GRBIO registration completed; waiting on web form for Arctos portal info

**Comment by dperriguey** [comment]
@ccicero @dbloom 
Our data is in Arctos and we will have our xml file to you shortly. Thanks!

**Comment by dperriguey** [comment]
@ccicero @dbloom IPT Metadata Created Data in Arctos
I have attached our xml file. I was unsure of what I should do with boundingCoordinates and dateStamp, but everything else should be good to go. It wouldn't let me link 

[eml-eps_paleo-v1.0.zip](https://github.com/ArctosDB/new-collections/files/1753068/eml-eps_paleo-v1.0.zip)

**Comment by campmlc** [comment]
Thanks, Dustin. Are you going to be loading more records before this is
published, or do you want it published as soon as possible?
Mariel

**Comment by dperriguey** [comment]
@campmlc 
Maybe I'm a little confused about how this works. I will be uploading more over time. I will be putting in more next week for sure. Are we supposed to publish every time we want our data in iDigBio?
-Dustin

**Comment by campmlc** [comment]
No, but I believe they publish once a month? Dave? It would be good to get
everything set up so you can continue to load records.

**Comment by dperriguey** [comment]
Ok, so I just upload like normal to Arctos and every month our iDigBio gets updated? If that's the case, that should be fine.

**Comment by campmlc** [comment]
Yes, that is how it should be set up. Dave, let us know when this has been
published.
Thanks!

**Comment by dperriguey** [comment]
Is there a way to change things in the xml file? Say in the future we get a new collection manager or the title changes?

**Comment by dbloom** [comment]
In order to publish this resource to the IPT I need to know where it should live.  At the moment we have an organization with GBIF for the MSB, but not one for UNM or EPS.  If this collection needs to be registered under UNM or under EPS then I need to know who the primary contact for this new organization should be (don't worry - there is little to no responsibility here, beyond a few emails each year, and all this can change later, if needed).

So - which inst./dept./org are you going to use AND who is the primary contact for said group?

I have the metadata resource set already.  Links to the Arctos dbase are next.

**Comment by dbloom** [comment]
@dperriguey You can change content in the xml file at any time, but you'll need to alert me that you've made changes and get me the new file so I can update the IPT.  Short of that - especially if the changes are minor - it is probably more efficient to tell me or @ccicero what changes you need made and I can then implement them directly into the IPT for you and republish.

**Comment by dbloom** [comment]
Last question.

I see that you've got these data under the CC-CY-NC license.  Are you sure you want that level or restriction placed on these data?  I recommend you review the [VertNet Guide to Copyright and Licenses for Data Publishing](http://vertnet.org/resources/datalicensingguide.html) before this goes live.  We recommend the CC0 designation for several reasons - all of which are explained in the Guide - but mostly because occurrence data is pretty much fact and thus, not subject to copyright in the U.S. If you have imagery or other media, those contents can be protected under other, more restrictive CC designations.

Let me what you decide.

**Comment by dbloom** [comment]
OK, the resource has been created and linked and is ready to go, pending the questions asked above (or previously, if you're reading this in your email).

**Comment by dperriguey** [comment]
@dbloom Thank you for your help! I need to ask Cori what she wants for the licensing and the primary contact. I will get back to you after 12 pm mountain time after I meet with her.

**Comment by cemyers42** [comment]
@dbloom Thanks for your help with this! Let's do a new one under UNM Paleontology Collections (UNMP). I'm the primary contact: Corinne Myers, cemyers@unm.edu

Regarding the licensing, we are a little bit unclear how to proceed. The Paleontological Resources Preservation Act (http://vertpaleo.org/The-Society/Advocacy/Paleontological-Resources-Preservation-Act.aspx), requires confidentiality of exact locality data for all specimens collected on public lands (which many of our specimens are). See below passage:

SEC. 6309. CONFIDENTIALITY.
Information concerning the nature and specific location of a paleontological resource shall be exempt from disclosure under section 552 of title 5, United States Code, and any other law unless the Secretary determines that disclosure would--
(1) further the purposes of this subtitle;
(2) not create risk of harm to or theft or destruction of the resource or the site containing the resource; and
(3) be in accordance with other applicable laws.

As long as having a CC0 license does not put us in noncompliance with this set of federal regulations, we are happy to adopt it. Thanks for your thoughts on this!

**Comment by dustymc** [comment]
@cemyers42 only unencumbered data are made available to/via DWC. Arctos offers various encumbrances to mask locality data from public users - http://arctos.database.museum/info/ctDocumentation.cfm?table=CTENCUMBRANCE_ACTION.

The DWC license governs what folks can do with data they access via DWC, not what is made available.

There's a similar license in Arctos (under manage collection) which governs what folks can do with data they access from Arctos.

http://handbook.arctosdb.org/documentation/encumbrance.html

**Comment by dperriguey** [comment]
@dbloom Ok, we're comfortable with the CC0 license.

**Comment by dbloom** [comment]
Super.  Thanks everyone.  I will begin the process to register a new "institution" with GBIF.

**Comment by dperriguey** [comment]
@dbloom Thank you for all your and everyone's help!

**Comment by dperriguey** [comment]
@dbloom do you know how long it takes for the registration to complete and when it all gets pushed to iDigBio?

**Comment by dperriguey** [comment]
@dbloom ok, so we see the published institution to GBIF. Thank you! Do you know when or how the data moves from Vertnet to GBIF to iDigBio? How do we actually get the data to publish?

**Comment by Jegelewicz** [comment]
MOU draft sent to @campmlc for review before sending to collection contact.
